### DISCUSSION TITLE
Get voltage, current  and power data

### DISCUSSION THREAD

**Discussion by MrRomo** [skydiver/ewelink-api#193] [Q&A]
Hi, there is an option to get voltage, current and power in real time through this API?

**Comment by alexose** [comment]
Hi @MrRomo,

It seems like Ewelink (aka coolkit) introduced a bunch of new functionality recently.  Doesn't look like this code supports it yet.

I see that you've started a fork off of the `release/v4.0.0` branch!  Cool.  I will hack on this a bit as well and perhaps we can convince @skydiver to get the latest changes merged in when we're ready.

**Comment by MrRomo** [comment]
Yeah, i was worked on a personal application for my house management and forked @skydiver lastest commit for build a custom client and works nice. Coolkit just added  new properties like dayKwh, dayTimeWork, reactive and aparent power, but this just work on lastest version of their products like the new S40.
![Screenshot from 2023-01-10 10-52-40](https://user-images.githubusercontent.com/40845483/211600023-818f9e3c-279a-4998-b984-f6e630c6187f.png)
